pdq/php/pdqhashertest.php

```diff
@@ -14,13 +14,18 @@
 //$show_timings = true;
 $dump = false;
 
+$downsample = false;
+
 foreach ($filenames as $filename) {
 
-  list ($hash, $quality) = PDQHasher::computeHashAndQualityFromFilename($filename, $show_timings, $dump);
+  list ($hash, $quality) = PDQHasher::computeHashAndQualityFromFilename($filename, $show_timings, $dump, $downsample);
```

Comment:
Should we add a test here where we also do not downsample?

We might not get the exact same hash, but we should get a hash within a small tolerance?